front/planning.php

```diff
@@ -37,7 +37,9 @@
 
 include ('../inc/includes.php');
 
-Session::checkRight("planning", READ);
+if (!isset($_GET['genical'])) {
```

Comment:
genical action checks user identity by a 'token' GET parameter.
I removed session check cause in case on ical export, we couldn't have any session (from a mail client for example)